guides/common/modules/proc_creating-an-activation-key.adoc

```diff
@@ -17,13 +13,12 @@ Subscription Manager attaches subscriptions differently depending on the followi
 Note that {Project} automatically attaches subscriptions only for the products installed on a host.
 For subscriptions that do not list products installed on {RHEL} by default, such as the Extended Update Support (EUS) subscription, use an activation key specifying the required subscriptions and with the auto-attach disabled.
```

Comment:
GitHub won't let me reference the lines above, but this entire section should be removed:

```
* Are there any subscriptions associated with the activation key?
* Is the auto-attach option enabled?
* For {RHEL} 8 hosts: Is there system purpose set on the activation key?
Note that {Project} automatically attaches subscriptions only for the products installed on a host.
For subscriptions that do not list products installed on {RHEL} by default, such as the Extended Update Support (EUS) subscription, use an activation key specifying the required subscriptions and with the auto-attach disabled.
```